proseco/catalog.py

```diff
@@ -153,7 +155,7 @@ def _get_aca_catalog(**kwargs):
     aca.version = VERSION
 
     # Remove kwargs that are specific to AcaTable
-    for kwarg in ('t_ccd', 't_ccd_penalty_limit'):
+    for kwarg in ('t_ccd', 't_ccd_penalty_limit', 'duration', 'target_name'):
```

Comment:
There's also some kwarg popping down a bit from this... could that be condensed?

Reply:
Done.